_posts/zh/newsletters/2023-07-26-newsletter.md

```diff
@@ -0,0 +1,89 @@
+  Andrew Poelstra 给出了手动计算验证技术（比如 [codex32][news239 codex32]）的概述，然后逐步分解如何用纸笔从一个私钥中推算出其公钥，他估计这个过程会花费至少 1500 小时（即使优化了流程也是如此）。{% assign timestamp="57:18" %}
+
+- [<!--why-are-there-17-native-segwit-versions-->为什么隔离见证会有 17 种原生版本？]({{bse}}118974)
+  Murch 解释道，[隔离见证][topic segwit]为[见证脚本的版本][bip141 witness program]定义了 17 个值（0 - 16），这是因为[比特币 Script][wiki script] 的常数操作码只有 17 个可用（OP_0 ... OP_16）。他指出，其他的数字需要使用数据效率更低的 `OP_PUSHDATA` 操作码实现。{% assign timestamp="59:43" %}
+
+- [<!--does-0-opcsv-force-the-spending-transaction-to-signal-bip125-replaceability-->`0 OP_CSV` 是否强制让花费交易使用 BIP125 可替换性信号？]({{bse}}115586)
+  Murch 指出一场[讨论][rbf csv discussion]确认了，因为 `OP_CHECKSEQUENCEVERIFY`（CSV）[时间锁][topic timelocks]和手续费替换（[RBF][topic rbf]）都使用 `nSequence` 字段来[<!--enforced-->实现]({{bse}}87376)，所以一个带有 `0 OP_CSV` 的输出将要求花费交易使用 [BIP125][] 可替换性信号。{% assign timestamp="1:03:04" %}
+
+- [<!--how-do-route-hints-affect-pathfinding-->“路由提示” 会如何影响寻路？]({{bse}}118755)
+  Christian Decker 解释了闪电支付的接收方想给发送方提供路由提示的两个原因。一个原因是接收者希望使用[未公开的通道][topic unannounced channels]，需要提示来帮助找出路径。另一个原因是，给发送者提供一个具有充足余额的通道的列表，以完成支付；他将这种技术称为 “加速路由”。{% assign timestamp="1:08:23" %}
+
+- [<!--what-does-it-mean-that-the-security-of-256bit-ecdsa-and-therefore-bitcoin-keys-is-128-bits-->256 位的 ECDSA 的安全性如何？为什么比特币的公钥是 128 位的？]({{bse}}118928)
+  Pieter Wuille 解释道，因为有算法可以比暴力搜索更高效地从公钥中推导出私钥，所以 256 位的 ECDSA 只能提供 128 位的安全性。他接着指出纯密钥安全性与[种子][topic bip32]安全性的区别。{% assign timestamp="1:12:26" %}
+
+## 新版本和候选版本
+
+*热门的比特币基础设施项目的新版本和候选版本。请考虑升级到新版本或帮助测试候选版本。*
+
+- [HWI 2.3.0][] 是这个让软件钱包可以跟硬件签名设备沟通的中间件的新版本。它添加了对 DIY 的 Jade 设备的支持以及一个二进制文件，可以在安装了 MacOS 12.0+ 的 Apple Silicon 硬件上运行 `hwi` 主程序。{% assign timestamp="1:15:09" %}
+
+- [LDK 0.0.116][] 是这个用于创建闪电网络赋能的应用的库的新版本。它包含了对 “[锚点输出][topic anchor outputs]” 和使用 [keysend][topic spontaneous payments] 的 [多路径支付][topic multipath payments] 的支持。{% assign timestamp="1:16:37" %}
+
+## 重大的代码和文档变更
+
+*本周出现重大变更的有：[Bitcoin Core][bitcoin core repo]、[Core Lightning][core lightning repo]、[Eclair][eclair repo]、[LDK][ldk repo]、[LND][lnd repo]、[libsecp256k1][libsecp256k1 repo]、[Hardware Wallet Interface (HWI)][hwi repo]、[Rust Bitcoin][rust bitcoin repo]、[BTCPay Server][btcpay server repo]、[BDK][bdk repo]、[Bitcoin Improvement Proposals (BIPs)][bips repo]、[Lightning BOLTs][bolts repo] 和 [Bitcoin Inquisition][bitcoin inquisition repo]。*
+
+- [Bitcoin Core GUI #740][] 升级了 [PSBT][topic psbt] 操作对话，以将给你自己的钱包支付的输出标记为 “你自己的地址”。这简化了对导入的 PSBT 的结果的评估，尤其是将找零返回给发送者的交易。{% assign timestamp="1:17:18" %}
+
+
+{% include references.md %}
+{% include linkers/issues.md v=2 issues="740,1096" %}
+[russell closing]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-July/004013.html
+[kc notes]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2023-July/004014.html
+[news193 gossip]: /en/newsletters/2022/03/30/#continued-discussion-about-updated-ln-gossip-protocol
+[news204 gossip]: /en/newsletters/2022/06/15/#gossip-network-updates
```

Comment:
It looks like this one has a Chinese translation, so perhaps you could link to that instead:
```suggestion
[news204 gossip]: /zh/newsletters/2022/06/15/#gossip-network-updates
```